cl/lib/model_helpers.py

```diff
@@ -562,3 +562,46 @@ def linkify_orig_docket_number(agency: str, og_docket_number: str) -> str:
     """
     # If no match is found, return empty str
     return ""
+
+
+class CSVExportMixin:
+
+    def get_csv_columns(self, get_column_name: bool = False) -> List[str]:
+        """Get list of column names required in a csv file.
+        If get column name is True. It will add class name to attribute
+
+        :param: get_column_name: bool. Whether add class name to attr name
+
+        :return: list of attrs of class to get into csv file"""
+        raise NotImplementedError(
+            "Subclass must implement get_csv_columns method"
+        )
+
+    def get_column_function(self) -> Dict[str, Callable[[str], str]]:
+        """Get dict of attrs: function to apply on field value if it needs
+        to be pre-processed before being add to csv
+
+        returns: dict -- > {attr1: function}"""
+        raise NotImplementedError(
+            "Subclass must implement get_column_fuction method"
+        )
+
+    def to_csv_row(self) -> List[str]:
```

Comment:
From Python 3.9 ([PEP 585](https://www.python.org/dev/peps/pep-0585/)) onwards `dict`, `list` and various other classes are now generic types. Using these rather than their typing counterpart is now preferred. From Python 3.9 you can now just do:

```suggestion
    def to_csv_row(self) -> list[str]:
```